file/file.cpp

```diff
@@ -21,7 +21,7 @@ File::File(const QString& newName,
 }
 
 const std::vector<QString>& getSupportedExtensions() {
-    static const std::vector<QString> SUPPORTED_EXTENSIONS = {"Изображения (*.png *.jpg)",
+    static const std::vector<QString> SUPPORTED_EXTENSIONS = {"Изображения (*.png *.jpg *.jpeg)",
                                                        "Текст (*.txt)"};
```

Comment:
Проверить строчку

Reply:
<img width="1212" alt="Снимок экрана 2021-10-05 в 20 29 46" src="https://user-images.githubusercontent.com/38105548/136073286-74628bbd-edfc-41cc-9880-735db07901ac.png">
перевод строки там есть, подвину чтобы они были друг под другом

```suggestion
    static const std::vector<QString> SUPPORTED_EXTENSIONS = {"Изображения (*.png *.jpg *.jpeg)",
                                                                                                                  "Текст (*.txt)"};
```